opsworks_deploy_python/templates/default/supervisor_upstart.erb

```diff
@@ -8,8 +8,12 @@ setuid <%= @user %>
 setgid <%= @group %>
 <% if @environment %>
 <% @environment.each do |key,val| %>
+<% if val =~ /\s/ %>
+env <%= key %>=<%= "\"#{val}\"" %>
```

Comment:
In the interest of simplicity, why not just replace line 11 with this, and omit the conditional?

Reply:
let me test that in a few